src/add-ns/index.ts

```diff
@@ -362,15 +362,15 @@ const addDependencies = () => (tree: Tree, context: SchematicContext) => {
 
   // @UPGRADE: Update all versions whenever {N} version updates
   const depsToAdd = {
-    'nativescript-angular': '~8.0.1',
+    'nativescript-angular': '~8.2.0',
     'nativescript-theme-core': '~1.0.4',
     'reflect-metadata': '~0.1.12',
-    'tns-core-modules': '~6.0.0',
+    'tns-core-modules': '~6.1.0',
   };
   packageJson.dependencies = {...depsToAdd, ...packageJson.dependencies};
 
   const devDepsToAdd = {
-    'nativescript-dev-webpack': '~1.0.0',
+    'nativescript-dev-webpack': '~1.2.0',
     '@nativescript/schematics': '~0.7.0',
     '@nativescript/tslint-rules': '~0.0.2',
```

Comment:
It seems it should be updated as well as it is updated here - https://github.com/NativeScript/nativescript-schematics/pull/260/files#diff-088c37f4fbd56afe5558316a0b613913R43

Reply:
you are right, I've missed that.

Reply:
Actually, we checked that and it is not a problem as both `'@nativescript/tslint-rules': '~0.0.2'` and `'@nativescript/tslint-rules': '~0.0.4'` installs `@nativescript/tslint-rules@0.0.5`

Reply:
honestly, that was purely for visual consistency ;)